token/transport.go

```diff
@@ -251,6 +251,7 @@ func DecodeRemoteClaimsResponse(_ context.Context, response *http.Response) (int
 	if response.StatusCode < 200 || response.StatusCode > 299 {
 		err := &DecodeClaimsError{
 			StatusCode: response.StatusCode,
+			Err:        errors.New(string(body)),
```

Comment:
@johnabass I noticed some way of capturing the response from RBL might be useful for error handling

Reply:
This is almost what we want.  I don't think we want exactly this, as we need a more holistic way of working with remote errors.

Reply:
Okay - I'll focus this PR on local development and documentation. We can address that on a later PR.